nixos/modules/config/shells-environment.nix

```diff
@@ -172,6 +172,13 @@ in
          export PATH="$HOME/bin:$PATH"
        '';
```

Comment:
nit: why here (between `system.build.setEnvironment` and `system.activationScripts.binsh`), not in the `environment.` section above?

Reply:
No good reason. I'll go ahead and move it.